Gems/Atom/RHI/Code/Include/Atom/RHI/CpuProfiler.h

```diff
@@ -39,31 +40,16 @@ namespace AZ
             };
 
             CachedTimeRegion() = default;
-            CachedTimeRegion(const GroupRegionName* groupRegionName);
-            CachedTimeRegion(const GroupRegionName* groupRegionName, uint16_t stackDepth, uint64_t startTick, uint64_t endTick);
+            CachedTimeRegion(const GroupRegionName& groupRegionName);
```

Comment:
Nitpick: Should probably be marked with `explicit`.

Reply:
Noted for the follow up work.